context/context.go

```diff
@@ -30,6 +32,8 @@ import (
 type AptlyContext struct {
 	sync.Mutex
 
+	ctx gocontext.Context
```

Comment:
I was just browsing through this change. Thanks for working on this issue.

One question I have is though wouldn't it be better and more readable when go context is an embedded type of Aptly context (like sync.Mutex)? This way aptly and go context can be used interchangeably.

Reply:
Interesting idea, I never thought about it to be honest. I'll try it, thanks!